app/serializers/concerns/user_auth_tokens_mixin.rb

```diff
@@ -22,10 +22,8 @@ def client_ip
   def location
     ipinfo = DiscourseIpInfo.get(client_ip, locale: I18n.locale)
 
-    location = [ipinfo[:city], ipinfo[:region], ipinfo[:country]].reject { |x| x.blank? }.join(", ")
-    return I18n.t('staff_action_logs.unknown') if location.blank?
-
-    location
+    return I18n.t('staff_action_logs.unknown') if ipinfo[:location].blank?
```

Comment:
These two lines can be shortened to this ;)

```suggestion
    ipinfo[:location].presence || I18n.t("staff_action_logs.unknown")
```